src/figuresClasses.ts

```diff
@@ -1,19 +1,79 @@
 export interface Figure {
-
+  color: string,
+  getArea(): number,
+  a?: number,
```

Comment:
Remove optional properties a, b, c, radius, width, and height from the Figure interface, as these properties should be specific to the respective classes.